src/main.cpp

```diff
@@ -1,4 +1,10 @@
+#ifndef __SDL2__FOLDER_MISSING__
 #include "SDL2/SDL.h"
+#else
+#include "SDL.h"
+#endif
```

Comment:
What's going on here? 🤔

Reply:
some sdl / sdl2 fun, if you use packaged (either as deb for ubuntu with .so s and includes, or dll and includes) sdl2, it comes with the include folders sdl2/SDL_*.h, but the source from sdl2 is just a plain old include folder (here)[https://github.com/libsdl-org/SDL/tree/SDL2/include] since SDL doesn't distinguish between sdl2 in it's source tree, it's a semver named version

so if you compile it from source there is no folder SDL2/ with the header files but the header files are just in the "include" folder :sweat_smile: 

There are better solutions, but this is the simplest one, some possible solutions:
- always use #include "SDL.h" and it would have no drawback, so thats better, why haven't I thought of that :facepalm: 

here are the reason why it's ALWAYS fine:
with cmake it pulls the SDL2 dependency and links with that, no clash with sdl1 possible
in meson it either detects the SDL2 dependency, or builds SDL2 from source, in both times it can't conflict with SDL1